src/NUnitEngine/nunit.engine.tests/Services/ResultWriters/XmlTransformResultWriterTests.cs

```diff
@@ -48,6 +48,13 @@ public void SummaryTransformTest()
 
             Assert.That(output, Contains.Substring(summary));
         }
+
+        [Test]
+        public void XmlTransformResultWriterIgnoresDTDs()
+        {
+            var transformPath = GetLocalPath("TransformWithDTD.xslt");
+            Assert.DoesNotThrow(() => new XmlTransformResultWriter(new object[] { transformPath }));
```

Comment:
Intrigued - does this not flag up a warning with the classic assert analysers? (As opposed to `Assert.That(() => {}, Throws.Nothing)

Reply:
Currently, there we only handle `Assert.IsTrue`, `Assert.True`, `Assert.IsFalse`, `Assert.False`, `Assert.AreEqual`, and `Assert.AreNotEqual` (see e.g. https://github.com/nunit/nunit.analyzers/blob/11b30aff05f0652d8697dc34f3faf0aa7ddd9813/src/nunit.analyzers/ClassicModelAssertUsage/ClassicModelAssertUsageAnalyzer.cs#L20-L25), but we could make an issue for `Assert.Throws` and `Assert.DoesNotThrow` - even though `Assert.Throws` can be useful as it allow us to capture the exception for further testing.

Reply:
Cool, no worries then!